### PR TITLE
add operations section

### PR DESCRIPTION
**Pull request by johnbister**
adding operations icon section

### CODE REVIEW

**Comment by johnbister** [comment]
Note: This PR is pretty messy and some styling still needs to be done, but I want to make sure we get the operations section up to make it easier for people to add their icons.

**Review by kristastarr — COMMENTED** [review]
The code looks great! 🔥 Love the animations as well, so I'm good with merging based on that, but I made a comment about the spacing between the sections - lmk what you think!

File: example/src/app.scss [hunk 1]
```diff
@@ -32,7 +32,7 @@ h3 {
 }
 
 .bx--grid {
-	margin: 0 auto 320px auto;
+	margin: 0 auto 160px auto;
```
[the remark this answers is not in the document]
Comment (johnbister) [reply example/src/app.scss:35]: That sounds great! Let's take a look at the Carbon site and see what specs they use and figure out what classes we should add. Thanks Krista!

**Review by kristastarr — APPROVED** [review]
Based on our slack conversation, I'll go ahead and merge this and we can work on the styling later! 👍

**Pull request merged**